### PR TITLE
Update metrics that missed labels

### PR DESCRIPTION
**Pull request by opan**
Update for PR #857

### CODE REVIEW

**Comment by opan** [comment]
This update is required. Without the labels, the metrics won't show up in `/metrics`. I have tested this locally. I will try to add some tests for this.

**Comment by oliver006** [comment]
Thanks.
PR looks good but this time let's add tests to make sure the metrics really get recorded.

**Comment by opan** [comment]
I have try to add the test in  [here](https://github.com/oliver006/redis_exporter/pull/866/commits/863ca08c5a90941ff726ead9843fc83713df8bf8).

If this does not suffice, I will try to add more test regarding this.

**Comment by oliver006** [comment]
Ok, thanks, let me have another look.

File: contrib/docker-compose-for-tests.yml [hunk 1]
```diff
@@ -32,24 +32,28 @@ services:
       - "16390:6390"
 
   redis-2-8:
+    platform: linux/amd64
```
Comment (oliver006) [inline contrib/docker-compose-for-tests.yml:35]: What are these all about?

**Review by oliver006 — APPROVED** [review]
PR looks good - just one Q about the docker-compose changes.

File: contrib/docker-compose-for-tests.yml (hunk 1 above)
Comment (oliver006) [reply contrib/docker-compose-for-tests.yml:35]: Can we revert this? let's keep the docker-compose file platform-agnostic.

File: contrib/docker-compose-for-tests.yml [hunk 2]
```diff
@@ -32,24 +32,28 @@ services:
...
     image: redis:2.8
     command: "redis-server --port 6381 --dbfilename dump2-8.rdb"
     ports:
       - "16381:6381"
 
   keydb-01:
+    platform: linux/amd64
```
Comment (oliver006) [inline contrib/docker-compose-for-tests.yml:42]: Revert these plz

**Review by oliver006 — CHANGES_REQUESTED** [review]

File: contrib/docker-compose-for-tests.yml (hunk 1 above)
Comment (opan) [reply contrib/docker-compose-for-tests.yml:35]: Ah yes, I am running the test on M1 Apple silicone machine. There are some warnings shown and that's why I put it there. Sure, I will remove it

Comment (oliver006) [reply contrib/docker-compose-for-tests.yml]: Thanks!

**Pull request merged**